### DISCUSSION TITLE
total newbie questions about products and scripts

### DISCUSSION THREAD

**Discussion by mcdonaldzachary** [bigcommerce/catalyst#1076] [Q&A] [answered]
Hello and pardon my ignorance i didn't want to try an install and screw up my current site. I read the documentation and upon doing the npm install you'd login to bigcommerce and create a new store front, does this mean it will recognize my current products? 
Also will current storefront scripts effect the catalyst version of my site?
thank you

**Comment by BC-krasnoshapka** [comment, marked as answer]
hi @mcdonaldzachary 

- if you create new Catalyst channel (as described in the [doc](https://www.catalyst.dev/docs/cli#create-a-new-catalyst-project)) you'll be able to assign existing products to the new channel later
- your existing scripts won't affect Catalyst instance as they're not loaded in Catalyst

**Reply by mcdonaldzachary** [reply]
Thank you for the reply that helps a lot. I'll be sure to read that doc
